dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -815,7 +820,18 @@ private void reportOnRateLimit(String id, GHRateLimit startRateLimit, GHRateLimi
         }
     }
 
-    private GHRateLimit getGhRateLimitQuietly() {
+    public void reportOnGitHubRelease(GHRateLimit startRateLimit, GHRateLimit endRateLimit, String repository, String username, String gitReference, boolean isSuccessful) {
```

Comment:
Question that doesn't block the PR, what does the username even mean in the context of a github app install?
I'm guessing this is just the user than the lambda is using, but if I understand correctly (and I very well might not), this user should be irrelevant for the rate limit on a particular installation

Reply:
Yeah it's the user who triggered the endpoint and they may or may not have a Dockstore user, but yeah, it's not relevant to the rate limit.

Reply:
Agreed it's not relevant to the rate limit, but [this comment](https://github.com/dockstore/dockstore/blob/fb1ab12875b7359b823b09043defa632b17c0f62/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java#L319) says it will fail if the user isn't in Dockstore.

Just speculating, beyond the scope of this PR, I wonder if we are missing some updates because of this? Presumably the gatk GitHub repo has GitHub users who aren't Dockstore users. I'm going to create a follow-up ticket.

Reply:
The last argument is `allowFail = false`, so it should pass.

Reply:
I misunderstood. We do get updates to existing workflows if it's a non-existent Dockstore user. We just don't create new workflows for a non-existent users. Not going to create a followup ticket.